### PR TITLE
Complete 4. get-article-by-id and error handling with tests

### PR DESCRIPTION
**Pull request by yhdchan**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -26,6 +26,33 @@ describe('GET /api/topics', () => {
 	})
 })
 
+describe('GET /api/articles/:article_id', () => {
+	test('200: should return an article object which should have the properties including author, title, article_id, body, topic, created_at and votes',() => {
+		return request(app)
+			.get('/api/articles/1')
+			.expect(200)
+			.then(({ body }) => {
+				expect(body.article).toMatchObject({
```
Comment (yalicee) [inline __tests__/app.test.js]: The expected keys should have correct values. They must be values related to the requested article e.g. article_id: 1, NOT article_id: expect.any(Number). Same for all the other properties

**Review by yalicee — CHANGES_REQUESTED** [review]
See my comment. Otherwise looks good :)

File: __tests__/app.test.js (hunk 1 above)
Comment (yhdchan) [reply __tests__/app.test.js]: Okay. I see. Thanks. I mixed them up

**Review by yalicee — APPROVED** [review]
👍

**Pull request merged**